src/map/modifier.h

```diff
@@ -195,6 +195,7 @@ enum class Mod
     PENGUIN_RING_EFFECT   = 152, // +2 on fishing arrow delay / fish movement for mini - game
     ALBATROSS_RING_EFFECT = 153, // adds 30 seconds to mini - game time
     PELICAN_RING_EFFECT   = 154, // adds extra skillup roll for fishing
+    FISHING_SKILL_GAIN    = 155, // food for fishing skill ups
```

Comment:
Please update the comments in this file to reflect that this spare is now used (at the bottom of this table)

Reply:
aye. thank you!